### ISSUE TITLE
Possible problem when utilizing min and max of properties

### ISSUE THREAD

**Issue by linusha** [LivelyKernel/lively.next#181]
We had a property that was defined like this:

```
_progress: {
        isFloat: true,
        min: 0,
        max: 1
      }
```

The only code that set this property was ` this._progress = (scrollPosition - this.start) / this.duration;`

However, we got e.g., negative values for _progress. 

Is this something that is wrong with lively or did we use this feature wrong?

**Comment by merryman** [comment]
Yes, right now `min` and `max` just as `isFloat` incidentially, are just interpreted by the inspector. We can however go further and start instrumenting the setter itself. The right module to implement this behaviour would be https://github.com/LivelyKernel/lively.next/blob/master/lively.classes/properties.js

**Comment by linusha** [comment]
This should eventually allow for some kind of type-checking, to make things like #884 unnecessary.
